cluster/crds/pkg.crossplane.io_deploymentruntimeconfigs.yaml

```diff
@@ -4312,8 +4313,6 @@ spec:
                                         More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
                                         TODO: Drop `kubebuilder:default` when controller-gen doesn't need it https://github.com/kubernetes-sigs/kubebuilder/issues/3896.
                                       type: string
-                                  required:
-                                  - name
```

Comment:
Is this related to the rest of the PR? If not, could you do it in a separate commit?

Reply:
No, I think I messed something up with the generator. I removed it.